### ISSUE TITLE
Question on kaiju classification from PacBio data

### ISSUE THREAD

**Issue by DaRinker** [bioinformatics-centre/kaiju#246]
I would appreciate some insight/confirmation that Kaiju is operating as expected.

I just happed to spot-check a 6.6kb PacBio read that was classified by kaiju (using 'nr_euk' DB). The kaiju-assigned taxon (txid 36630) doesn't appear among the 100 top-BLAST hits (NCBI nr).  The top blast hit (88% coverage with 99% sequence identity) corresponds to NCBI:txid5061 (a taxa that is present in the nr_euk)

Conversely, if I take another read that Kaiju has classified as belonging to txid 5061, it BLASTs very strongly (99% match over 7.7kb) to a different taxa (NCBI:txid746128).

Any ideas what might be going on? Might kaiju be having problems with these long reads? I'm now concerned that I cannot trust any of the classifications I'm getting back.

**Comment by pguenzi-tiberi** [comment]
Hello @DaRinker,

Have you continued your trials with Kaiju and long reads ? Did you manage to improve the classification scores ?

**Comment by DaRinker** [comment]
No. I decided it was the wrong tool for the job. Maybe there have been developments since? Right now I'm most confident when using Kaiju on Illumina reads.

**Comment by pguenzi-tiberi** [comment]
Thank you very much for your reply !!!
